test/js/prepare-worksheets_test.js

```diff
@@ -0,0 +1,29 @@
+var expect = require('chai').expect;
+var jsdom = require('mocha-jsdom');
+
+describe('Prepare worksheets', function() {
+
+  var $,
+      worksheets;
+
+  // Load in the HTML of the prepare-worksheets page. It'd be ideal to load the
+  // live content but Sheer makes that difficult.
```

Comment:
Normally we'd use something like [grunt-contrib-jasmine](https://github.com/gruntjs/grunt-contrib-jasmine) to automagically run our tests through PhantomJS but OAH can't do that because our templates are processed and served by Sheer. We have to first install and start Sheer to get the HTML of any OAH page. Locally, this isn't a problem but with external services like Travis, we'd have to write a script to get it up and running on their virtual machines. This would be difficult and it kind of defeats the purpose of unit tests which are supposed to test small, individual units of behavior.

The proper solution IMHO is to refactor the code to decouple it from the DOM. That way we can test methods' logic without requiring much HTML. The [dropdown tests](https://github.com/cfpb/owning-a-home/blob/master/test/js/dropdown-utils_test.js#L16) are a good example of this.

For now we're just pasting in a bunch of HTML and saying "Hey Mocha, pretend this huge blob of HTML is a webpage and run the prepare-worksheet tests against it".